lib/utils.js

```diff
@@ -81,9 +128,8 @@ function mergeQuery(base, update, spec) {
     if (!base.include) {
       base.include = update.include;
     } else {
-      var saved = base.include;
-      base.include = {};
-      base.include[update.include] = saved;
+      //merge inclusion
+      base.include = mergeIncludes(base.include, update.include);
```

Comment:
Please double check to make sure the `include` in default scope should have precedence.